test/repositories_include_test.rb

```diff
@@ -2,12 +2,25 @@
 
 require_relative "./test_helper"
 
+require "y2packager/dialogs/addon_selector"
+
 # just a wrapper class for the repositories_include.rb
-class RepositoryIncludeTester
-  extend Yast::I18n
-  Yast.include self, "packager/repositories_include.rb"
+module Yast
+  class RepositoryIncludeTesterClass < Module
+    extend Yast::I18n
+  end
+
+  def main
+    Yast.include self, "packager/repositories_include.rb"
```

Comment:
so you include to whole Yast namespace this repositories include? Why not include it to RepositoryIncludeTesterClass?